app/res/layout/nav_drawer_footer.xml

```diff
@@ -13,6 +13,35 @@
         android:layout_height="1dp"
         android:background="@color/darker_grey"/>
 
+    <LinearLayout
```

Comment:
curious if there are conversations/figma that talks about putting this in footer instead of a regular list item ? I think it makes sense but just want to make sure product team is aware of it.